core/chaincode/exectransaction.go

```diff
@@ -139,6 +140,9 @@ func ExecuteTransactions(ctxt context.Context, cname ChainName, xacts []*pb.Tran
 	txerrs = make([]error, len(xacts))
 	for i, t := range xacts {
 		_, txerrs[i] = Execute(ctxt, chain, t)
+		if txerrs[i] != nil {
+			sendTxRejectedEvent(xacts[i].Uuid, txerrs[i].Error())
```

Comment:
Do you mean somewhere in CommitTxBatch? (e.g. a for loop over all the erroneous TXs calling Send?) By the way this commit does not remove the storage of errored TXs.